.github/workflows/branchcleanup.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Branch Cleanup
+# This workflow is triggered on all closed pull requests.
+# However the script does not do anything it a merge was not performed.
+on:
+  pull_request:
+    types: [closed]
+
+env:
+  NO_BRANCH_DELETED_EXIT_CODE: 0
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jessfraz/branch-cleanup-action@master
```

Comment:
There is litteraly a tick box in the github UI fo this now.

Reply:
problem is you need admin access and they haven't exposed it to the API, which means terraform cannot do it for us.

This at least makes it more obvious to people as to how we are doing it and does not require an admin remembering something (We already have that with sponsers, I don't want to be the bottleneck to lots of things)